Microsoft.Toolkit.Uwp.UI.Controls.Core/DropShadowPanel/IAlphaMaskProvider.cs

```diff
@@ -0,0 +1,20 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Windows.UI.Composition;
+
+namespace Microsoft.Toolkit.Uwp.UI
```

Comment:
The physical file needs to move within the `Microsoft.Toolkit.Uwp.UI` assembly as well.

Reply:
Ah right, I'll move it there! Any suggestion on where to place it? None of the existing folders seems relevant, is it ok if I create one called DropShadowPanel (following the same pattern as AdvancedCollectionView)?

Reply:
I think just putting in the `Helpers` folder for now would be alright. It'll be in the root namespace still, so folder structure doesn't really matter, we can always move it later if we want without breaking anyone. Thanks!